### ISSUE TITLE
Cann't view file contents

### ISSUE THREAD

**Issue by Mr-ziqiang** [DonJayamanne/gitHistoryVSCode#236]
**output:**
```
git show c81dba1:C:/testProject/tutorial4/sample.txt  (completed in 0.335s)
fatal: Path 'C:/testProject/tutorial4/sample.txt' does not exist in 'c81dba1'
```

![111](https://user-images.githubusercontent.com/16486126/36084548-544c9bd0-0ff9-11e8-99f0-4fd8300322df.jpg)

**Comment by Mr-ziqiang** [comment]
looks like the driver is error.
my code is local **d:**\testProject\tutorial4\sample.txt, but your commond line is **c:**\testProject\tutorial4\sample.txt

**Comment by Orunnerde** [comment]
I got the same problem as you described. My code is in D: but it tried to allocate the file in C:

**Comment by zzhhbyt1** [comment]
Recently run into the same output. But my old file is on another laptop which lies actually in C: when I commit it. 
And after got the repo on my desktop lies in D: , I used both gitHistory and GitLens but they both split same error.

**Comment by RainThinks** [comment]
I'm a Mac user, and I have the same issue. Nothing happened after click 'view the contents'. No editor opened, no output either.

OS: macOS 10.13.6
VS Code: 1.25.1
Git History: 0.4.2

**Comment by ole1986** [comment]
This issue should be resolved with version >= 0.5.0 since we are using native git api to fetch file content.
